morango/sync/operations.py

```diff
@@ -1127,7 +1127,18 @@ class LegacyNetworkQueueOperation(NetworkLegacyNoOpMixin, NetworkOperation):
     Without ASYNC_OPERATIONS capability, the server will perform queuing during initialization
     """
 
-    pass
+    def handle(self, context):
+        """
+        :type context: NetworkSessionContext
+        """
+        self._assert(ASYNC_OPERATIONS not in context.capabilities)
+
+        # When pushing, this should occur after queuing locally, so we need to update the server
+        # with how many records we've queued for the push
+        if context.is_push:
```

Comment:
I'm guessing this is the case, but just to double-confirm: when it's a pull *from* here, the counts on the other side will be properly updated without this mechanism? In other words, what matters here is that it's a push, not that we're the sending side?

Also, relatedly, is there a mechanism in place such that this task is only executed on the client side, not the server side? I can't remember if that's handled via asserts or elsewhere.